import java.util.*;

class Solution {
    public String removeDuplicates(String s) {
        Stack<Character> st=new Stack<>();

        for(char ch:s.toCharArray()) {
            if(st.isEmpty()) {
                st.push(ch);
                continue;
            }

            if(st.peek()==ch) st.pop();
            else st.push(ch);
        }

        StringBuilder ans=new StringBuilder();

        for(char ch:st)
            ans.append(ch);     

        return ans.toString();
    }
}